### PR TITLE
docs: add building apisix Chinese doc.

### PR DESCRIPTION
**Pull request by hf400159**
### Description

add building apisix Chinese doc. sync https://github.com/apache/apisix/pull/7219/

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/building-apisix.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+apisix quit
+```
+
+`apisix stop` 命令会强制关闭 APISIX 并丢弃所有请求。
+
+```shell
+apisix stop
+```
+
+## 为 APISIX 构建 apisix-base
```
Comment (SylviaBABY) [inline docs/zh/latest/building-apisix.md:207]:
```suggestion
## 为 APISIX 构建 APISIX-Base
```

File: docs/zh/latest/building-apisix.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+
+APISIX 的一些特性需要在 OpenResty 中引入额外的 NGINX 模块。
+
+如果要使用这些功能，你就需要构建一个自定义的 OpenResty 发行版（apisix-base）。请参考 [apisix-build-tools](https://github.com/api7/apisix-build-tools) 配置你的构建环境并进行构建。
```
Comment (SylviaBABY) [inline docs/zh/latest/building-apisix.md:211]:
```suggestion
如果要使用这些功能，你就需要构建一个自定义的 OpenResty 发行版（APISIX-Base）。请参考 [apisix-build-tools](https://github.com/api7/apisix-build-tools) 配置你的构建环境并进行构建。
```

**Review by SylviaBABY — DISMISSED** [review]

**Comment by guoqqqi** [comment]
image
The licence in this file is not identified by CI 🤔

**Comment by hf400159** [comment]
> image The licence in this file is not identified by CI 🤔

@guoqqqi fixed, PTAL.

**Review by SylviaBABY — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**